release/nightly_tests/nightly_tests.yaml

```diff
@@ -72,6 +72,21 @@
     cluster:
       compute_template: shuffle/shuffle_compute_smoke.yaml  # Does not exist yet
 
+# Test multi-node, multi-GPU Ray SGD example.
+- name: sgd_gpu
+  owner:
+    mail: "amog@anyscale.com"
+    slack: "@tune-team"
+
+  cluster:
+    app-config: sgd_gpu/sgd_gpu_app_config.yaml
+    compute_template: sgd_gpu/sgd_gpu_compute.yaml
+
+  run:
+    timeout: 3000
+    prepare: python wait_cluster.py 2 600
+    script: python sgd_gpu/sgd_gpu_test.py
+
```

Comment:
Let's move this into a different file. The `nightly_tests` is misleading (these are the "core nightly tests") and we should probably rename them (I can do that later). Let's move this into a separate sgd tests file

Reply:
I created a new directory called "nightly_gpu". This can also be place where we can add Horovod on Ray or Ray Lightning tests.